src/BSD-1-Clause.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="BSD-1-Clause" name="BSD 1-clause License">
+    <crossRefs>
+      <crossRef>https://svnweb.freebsd.org/base/head/include/ifaddrs.h?revision=326823</crossRef>
+    </crossRefs>
+    <copyrightText>
+      <p>
+        Copyright (c)<alt name="copyright" match=".+"><year> <owner></alt>
+        All rights reserved.
```

Comment:
It's a bit strange that “All” looks like the start of a new sentence, but the previous line doesn't end in a period or other terminal punctuation.  Looking [upstream][1], the copyright owner ends with “Inc.”, so I think we want:

```xml
Copyright (c)<alt name="copyright" match=".+"><year> <owner></alt><optional>.</optional>
```

to cover cases where the owner ends with a period (and you don't want to double up) as well as cases where the owner does not (so you need some additional terminal punctuation).
[1]: https://svnweb.freebsd.org/base/head/include/ifaddrs.h?revision=250887&view=markup